heron/config/src/yaml/conf/aurora/heron.docker.aurora

```diff
@@ -0,0 +1,50 @@
+"""
+Launch the topology as a single aurora job with multiple instances.
+The heron-executor is responsible for starting a tmaster (container 0)
+and regular stmgr/metricsmgr/instances (container index > 0).
+"""
+
+heron_core_release_uri = '{{CORE_PACKAGE_URI}}'
+heron_topology_image_uri = '{{TOPOLOGY_PACKAGE_URI}}'
+core_release_file = "heron-core.tar.gz"
+
+# --- processes ---
+fetch_heron_system = Process(
+  name = 'fetch_heron_system',
+  cmdline = 'curl %s -o %s && tar zxf %s' % (heron_core_release_uri, core_release_file, core_release_file)
```

Comment:
@chris-pardy - how does this work? If the docker image is supposed to contain both the core image as well as topology image, we need to just use docker pull to get the image, is it correct?